src/lib/workspace/DownloadTab/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import {
+  AnalysisState,
+  useConfiguredDownloadClient,
+  useStudyMetadata,
+} from '../../core';
+
+// Definitions
+import { EntityCounts } from '../../core/hooks/entityCounts';
+
+// Components
+import MySubset from './MySubset';
+import CurrentRelease from './CurrentRelease';
+
+// Hooks
+import { useStudyEntities, useWDKStudyReleases } from '../../core/hooks/study';
+import { useEnhancedEntityData } from './hooks/useEnhancedEntityData';
+import { DownloadTabStudyReleases } from './types';
+import PastRelease from './PastRelease';
+
+type DownloadsTabProps = {
+  analysisState: AnalysisState;
+  totalCounts: EntityCounts | undefined;
+  filteredCounts: EntityCounts | undefined;
+};
+
+export default function DownloadTab({
+  analysisState,
+  totalCounts,
+  filteredCounts,
+}: DownloadsTabProps) {
+  const downloadClient = useConfiguredDownloadClient('/eda-user-service');
```

Comment:
We'll want to avoid hardcoding the download client URL.

To do so, I would initialize the `downloadClient` in the workspace container and pass it down to the `DownloadTab` via props.

(An alternative would be to pass down the `downloadServiceUrl` via props, but passing the `downloadClient` itself is more consistent with how we're [initializing the other API clients](https://github.com/VEuPathDB/web-eda/blob/28f3a64fbf903525078d07a95bef9a95a4f32451/src/lib/workspace/WorkspaceContainer.tsx#L43-L45).)